### PR TITLE
Update `k8s.io/utils` to `9bab9ef40391`

### PR DESCRIPTION
**Pull request by ialidzhikov**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:
This PR updates the `k8s.io/utils` dependency to `9bab9ef40391` to adopt the following PRs:
- https://github.com/kubernetes/utils/pull/249
- https://github.com/kubernetes/utils/pull/251

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by dims** [comment]
@ialidzhikov please update to the latest HEAD of k8s.io/utils to pick up one more change that merged yesterday ( https://github.com/kubernetes/utils/compare/56c0de1e6f5e...HEAD )

thanks!

**Comment by ialidzhikov** [comment]
Sure, I will do it tomorrow morning.

**Comment by ialidzhikov** [comment]
@dims, the PR is ready for review now.

File: vendor/k8s.io/utils/exec/testing/hello [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```
Comment (liggitt) [inline vendor/k8s.io/utils/exec/testing/hello:1]: @dims - non-blocking for this PR, but might be worth a cleanup PR in utils to move this into a `testdata` folder so it doesn't get vendored unnecessarily

Comment (ialidzhikov) [reply vendor/k8s.io/utils/exec/testing/hello:1]: Nice suggestion, I will take it.

Comment (dims) [reply vendor/k8s.io/utils/exec/testing/hello:1]: 👍🏾

Comment (ialidzhikov) [reply vendor/k8s.io/utils/exec/testing/hello:1]: Ref https://github.com/kubernetes/utils/pull/252

**Pull request merged**